cypress/e2e/02_add_delete_node_page.cy.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference types="cypress" />
+require('cypress-xpath')
+describe("Verify Add and Delete node", () => {
+  // This method performs setup before each test.
+  beforeEach(() => {
+    // Open Eyes to start visual testing.
+    // Each test should open its own Eyes for its own snapshots.
+    cy.eyesOpen({
+      // The name of the application under test.
+      // All tests for the same app should share the same app name.
+      // Set this name wisely: Applitools features rely on a shared app name across tests.
+      appName: "studio",
+
+      // The name of the test case for the given application.
```

Comment:
i don't think that we need to keep all of these comments that were copy/pasted from the Applitools cypress tutorial?